tiles/src/main/java/com/protomaps/basemap/layers/Buildings.java

```diff
@@ -20,10 +21,33 @@ public String name() {
     return "buildings";
   }
 
+  static int quantize_val(double val, int step) {
+    // special case: if val is very small, we don't want it rounding to zero, so
+    // round the smallest values up to the first step.
+    if( val < step ) {
+      return (int) step;
+    }
+
+    return (int) Math.round(val / step) * step;
+  }
+
   @Override
   public void processFeature(SourceFeature sf, FeatureCollector features) {
-    if (sf.canBePolygon() && (sf.hasTag("building") || sf.hasTag("building:part"))) {
+    if (sf.canBePolygon() && (
+            ( sf.hasTag("building") && !sf.hasTag("building", "no")) ||
+            ( sf.hasTag("building:part") && !sf.hasTag("building:part", "no"))))
+    {
       Double height = parseDoubleOrNull(sf.getString("height"));
+      Double min_height = parseDoubleOrNull(sf.getString("min_height"));
+      Integer min_zoom = 11;
+      String kind = "building";
+
+      // Limit building:part features to later zooms
+      // TODO: (nvkelso 20230621) this should be based on area and volume, too
+      if( sf.hasTag("building:part") ) {
+        kind = "building_part";
+        min_zoom = 14;
```

Comment:
Building parts aren't useful to show until alter zooms when they become large enough on the map to discern.